rs/crypto/ecdsa_secp256r1/src/lib.rs

```diff
@@ -305,6 +434,57 @@ impl PrivateKey {
         let key = self.key.verifying_key();
         PublicKey { key: *key }
     }
+
+    /// Derive a private key from this private key using a derivation path
+    ///
+    /// This is the same derivation system used by the Internet Computer when
+    /// deriving subkeys for threshold ECDSA with secp256r1
+    ///
+    /// As long as each index of the derivation path is a 4-byte input with the highest
+    /// bit cleared, this derivation scheme matches SLIP-10
+    ///
+    /// See <https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-ecdsa_public_key>
+    /// for details on the derivation scheme.
```

Comment:
Opened https://dfinity.atlassian.net/browse/CRP-2627

Reply:
During discussion in the team meeting we realized updating the interface spec would not be appropriate since the IC itself does not currently support P-256 signatures. Instead this reference to the interface spec should be removed.

Reply:
Removed refs to the interface spec for the time being